src/trusted/service_runtime/nacl_globals.h

```diff
@@ -42,26 +42,12 @@ EXTERN_C_BEGIN
                      | ((uint64_t)(X)[6] << (6 * CHAR_BIT))	\
                      | ((uint64_t)(X)[7] << (7 * CHAR_BIT)))
 
-enum {
-  PIPE_NUM_MAX = 16,
-  CACHED_LIB_NUM_MAX = 32,
-  CAGING_LIB_PATH_MAX = 64,
-  FILE_DESC_MAX = 1024,
-  CHILD_NUM_MAX = 256,
-  CAGING_FD_NUM  = 256,
-  ARG_LIMIT = 4096,
-  PIPE_BUF_MAX = 4906
-};
+#define  CAGING_LIB_PATH_MAX  64
+#define  FILE_DESC_MAX        1024
+#define  CAGE_MAX             1024
+#define  CHILD_NUM_MAX        256
+#define  ARG_LIMIT            4096
```

Comment:
For clarity, these should ideally be written in hex or binary. I don't necessarily see the need to change it from an enum to defines, but this is just coding style and it can be done either way.

Reply:
I think these in decimal makes the most sense. Stylistically this made more sense to me as well.